client/src/components/summary/Summary.jsx

```diff
@@ -14,33 +14,41 @@ class Summary extends Component {
       <div styleName="island-summary">
```

Comment:
Maybe consider changing Summary from a class to a function since you're not really changing states